src/overlays/actors/ovl_En_Fall/z_en_fall.c

```diff
@@ -393,14 +425,17 @@ void EnFall_StoppedClosedMouthMoon_PerformCutsceneActions(EnFall* this, GlobalCo
                     func_800B9010(&this->actor, NA_SE_EV_FALL_POWER - SFX_FLAG);
                 }
                 break;
+
             case 1:
                 switch (globalCtx->csCtx.frames) {
                     case 561:
                         Audio_PlayActorSound2(&this->actor, NA_SE_EN_MOON_SCREAM1);
                         break;
+
                     case 590:
                         Audio_PlayActorSound2(&this->actor, NA_SE_EV_MOON_CRY);
                         break;
+
                     case 737:
                         Audio_PlayActorSound2(&this->actor, NA_SE_EV_SLIP_MOON);
                         break;
```

Comment:
I think it's probably okay to not have newlines in switches where the cases are this small, but perhaps let's just standardise to always.

Reply:
I'll just leave them as-is for now. I don't think they help readability in this case, but they don't hurt either. Unless someone disagrees with that, I'll just keep them the way they are to standardize things.